docs/known-issues.md

```diff
@@ -1,14 +1,14 @@
-| The Unity Editor doesn't correctly import navmeshes.                                                                                                                                                                                                                                                                         | 2018/10/12 | None                                                                                                   | Manually regenerate the navmesh after opening the project.            |
-| Running a worker that has been built using IL2CPP leads to a crash. This is due to [a bug in Unity](https://issuetracker.unity3d.com/issues/il2cpp-notsupportedexception-is-thrown-when-using-system-dot-type-dot-getcustomattributes-type-bool).                                                                                                         | 2019/01/28 | None                                                                                                | Use the Mono compiler. There is no workaround for iOS.                                                |
-| Running the FPS project in the Editor with the current platform as Android leads to the walls being slightly transparent.                                                                                                      | 2019/01/28 | None                                                                                                | It renders correctly when running in the Emulator or an Android device                                                |
-| Building a worker in the FPS starter project for Linux will throw errors when compiling the 'Hidden/PostProcessing/FinalPass' shader.                                                                                                    | 2019/01/28 | None                                                                                                | The build still succeeds and won't affect your worker.                                                |
-| Protocol logging on Linux workers is disabled due to a libcpp linking bug in Unity.                                                                                                   | 2019/01/28 | None                                                                                                | No workaround.                                                |
+| The Unity Editor doesn't correctly import NavMeshes.                                                                                                                                                                                                                                                                         | 2018/10/12 | None                                                                                                   | Manually regenerate the NavMesh after opening your project.            |
```

Comment:
I think the nav mesh one is gone as well!

Reply:
Correct

Reply:
@ElleEss ^^